packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -238,7 +238,7 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
           ActionButton(
             child: FlatButton(
               padding: EdgeInsets.zero,
-              child: const Icon(Icons.delete, size: 24.0),
+              child: const Icon(Icons.delete, size: actionsIconSize),
```

Comment:
Good change, apologies for adding a magic number there!

Reply:
No worries - we've been converting fixed constants to common, semantic ones in theme.dart over time.